### ISSUE TITLE
Add logging feature to check

### ISSUE THREAD

**Issue by simonmeggle** [simonmeggle/robotmk#48]

**Comment by m3rlinux** [comment]
Hi Simon,
I enabled syncronization of log file to CheckMK but I can't see the icon on service, see imege belove

![image](https://user-images.githubusercontent.com/16387701/153164512-ae9d730c-3f8d-4d74-9ad8-402d54f2d9b7.png)

The log files seem to be transmitted on CheckMK:
```
OMD[slave]:~/local/share/addons/robotmk$ tree 
.
|-- host01
|   |-- fae\ -\ Fae
|   |   |-- suite_last_error_log.html -> ../fae_last_error_log.html
|   |   `-- suite_last_log.html -> ../fae_last_log.html
|   |-- fae_last_error_log.html
|   |-- fae_last_log.html
`-- host01.local.domain
    |-- fae\ -\ Fae
    |   |-- suite_last_error_log.html -> ../fae_last_error_log.html
    |   `-- suite_last_log.html -> ../fae_last_log.html
    |-- fae_last_error_log.html
    |-- fae_last_log.html
```

Any suggestion?

**Comment by simonmeggle** [comment]
Hi, did you all the steps described in the inline help? (defining the icons, setting the rule for icons, ...)
I assume you are in a distributed setup (site "slave"). Unfortunately, the HTML log integration does not work on the master site. Do NOT try to synchronize (rync) the files up to the master site into `local/`. The master will then sync them to ALL slave sites (you wont't want that). 
=> On the slave site, do you see that the services got robotmk labels? 

To sum up, this approach was done as a kind of POC to show how an integration could look like. But the drawback is that anybody with cmk access will have access to all HTML logs (theoretically). 
Checkmk 2.1 will contain livestatus columns for Robotmk which allows to read the logs from a remote site. (see https://checkmk.com/werk/12701). This will mean that HTML logs are subjected to the same ACLs like other objects in Checkmk.

**Comment by m3rlinux** [comment]
Oops!
Many thanks Simon. I didn't see the inline help. Yes! I'm in a distributed setup but it would be nice access log by the slave gui only.
Following the guide provided in inline help I managed to have the two icons, but know the log files are disappeared from the filesystem on the slave (see belowe):
```
OMD[slave]:~/local/share/addons/robotmk$ tree 
.
|-- host01
|   |-- fae\ -\ Fae
|   |-- fae_last_log.html
`-- host01.local.domain
    |-- fae\ -\ Fae
    |-- fae_last_log.html
 ```
🤔

**Comment by simonmeggle** [comment]
This is most likely caused by the fact that I have chosen the wrong place for the logs. Sorry.  :-|
Each time you save configuration on the master, it synchronizes the contents below of local/share .... to the slaves (as well as  installed MKPs, local checks etc.). 
The better place would be under var/. This would require to change the apache config as well as the robotmk check. But it would allow to safely rsync the data to the master. 
Would you open an issue for that please? 
Thanks, 
Simon
